bentoml/server/gunicorn_server.py

```diff
@@ -17,11 +17,11 @@
 from typing import Optional
 
 import psutil
-from dependency_injector.wiring import Provide as P
+from dependency_injector.wiring import Provide
```

Comment:
nit: move to same line as wire.

Reply:
End up changing this back to avoid conflicts with other ongoing PRs